desc/coils.py

```diff
@@ -2487,7 +2489,12 @@ def to_FourierPlanar(
             minor radius r in a plane specified by a center position and normal vector.
 
         """
-        coils = [coil.to_FourierPlanar(N=N, grid=grid, basis=basis) for coil in self]
+        coils = [
+            coil.to_FourierPlanar(
+                N=N, grid=grid, basis=basis, check_intersection=check_intersection
```

Comment:
the base coils don't have a check_intersection option, just coilsets

Reply:
yes but a MixedCoilSet might contain CoilSets or other MixedCoilSets as its submembers, so we have to pass it here to this call, otherwise the submembers will check themselves for intersection when they convert